metadata-proxy/nginx.conf

```diff
@@ -0,0 +1,47 @@
+user www-data;
+worker_processes 4;
+pid /run/nginx.pid;
+
+events {
+  worker_connections 20;
+}
+
+http {
+  server {
+    listen 127.0.0.1:2020;
+
+    # By default, proxy everything.
+    location / {
+        proxy_pass http://169.254.169.254;
+    }
+
+    # Return 403 for anything too sensitive for pods. Would a regex be better?
```

Comment:
That's fair. I've switched to a whitelist. It makes a few things (e.g., REST discovery) weird, so I've aimed for clarity over brevity.

BTW, this config is overwritten by a configmap in the daemonset. I'm included this for the same reason as the "ENTRYPOINT": sane defaults. This allows users with sufficient permissions to overwrite the behavior.